hbase-server/src/test/java/org/apache/hadoop/hbase/util/TestRegionMover2.java

```diff
@@ -116,7 +116,7 @@ public void testWithMergedRegions() throws Exception {
       .collect(Collectors.toList());
     RegionMover.RegionMoverBuilder rmBuilder =
       new RegionMover.RegionMoverBuilder(rsName, TEST_UTIL.getConfiguration()).ack(true)
-        .maxthreads(8);
+        .maxthreads(2);
```

Comment:
Why this change?

Reply:
It is because now when moving a region at master side, we need locate meta first so we will go to master to ask the location for meta again. Now we only have 3 master rpc handlers so it is easy to dead lock if we have more than 3 parallel move requests...

Anyway, I've also changed this UT in #2392 and added comments to say this. Let me remove the modification in this PR.